src/hooks/useMultiSelect.stories.mdx

```diff
@@ -41,22 +39,134 @@ export default function MultiSelectWithHookExample() {
   const multiSelectId = "multi-select-1";
   // Without this hook, you'd have to add handlers for onChange and onClear,
   // and also manage the selectedItems state yourself.
-  const { onChange, onClear, selectedItems } = useMultiSelect(
-    multiSelectId,
-    items
+  const { onChange, onClear, selectedItems } = useMultiSelect();
+
+  return (
+    <MultiSelect
+      id={multiSelectId}
+      variant="listbox"
+      items={items}
+      selectedItems={selectedItems}
+      onChange={(selectedItem) => onChange(selectedItem.id, multiSelectId)}
+      onClear={() => {
+        onClear(multiSelectId);
+      }}
+    />
   );
+}
+```
+
+## useMultiSelect Hook implementation in a `dialog` MultiSelect
+
+```jsx
+import React from "react";
+import MultiSelect from "./MultiSelect";
+import useMultiSelect from "./MultiSelect/useMultiSelect";
```

Comment:
Change this to `import { MultiSelect, useMultiSelect } from "@nypl/design-system-react-components";`. This is an example for developers and it should show how _they_ should import it in their project. Update this on lines 35-36 as well.

Reply:
fixed. I also added this update to the MultiSelect.stories.mdx